no-restricted-imports.js

```diff
@@ -43,5 +43,10 @@ module.exports = {
 			message:
 				'Use this instead: import { isPageEditor } from "@/utils/expectContext";',
 		},
+		{
+			group: ["react-shadow/emotion"],
+			message:
+				'Use this instead: import EmotionShadowRoot from "@/components/EmotionShadowRoot"',
+		},
```

Comment:
This rule is pretty Extension specific. I'm not sure if it makes sense to include in the shared config.

Reply:
You're right, the app doesn't have any Shadow DOM. I'll move it back